packages/components/src/components/input-field/increment-buttons.tsx

```diff
@@ -79,7 +81,9 @@ const IncrementButtons = ({
         <React.Fragment>
             <Button
                 id={`${id}_add`}
-                className={'dc-input-wrapper__button dc-input-wrapper__button--increment'}
+                className={`dc-input-wrapper__button dc-input-wrapper__button--increment ${
+                    should_apply_disabled_style ? 'dc-input-wrapper__button--disabled' : ''
```

Comment:
The reason why `should_apply_disabled_style`  was added is that style which is applied inside of `<Button />` with the help of `is_disabled` is not enough.